app/models/user.rb

```diff
@@ -101,7 +101,7 @@ def agreement_letter_for_event(given_event)
   # @param [Event] given_event
   # @return [Boolean]
   def requires_agreement_letter_for_event?(given_event)
-    return self.older_than_required_age_at_start_date_of_event?(given_event, 18)
+    return (not self.older_than_required_age_at_start_date_of_event?(given_event, 18))
```

Comment:
IMO it does not, since when developing in Ruby you are used to the last statement in the function being the return value. In addition the [style guide](https://github.com/bbatsov/ruby-style-guide#no-explicit-return) defines it as a good practice. But I'll leave the final choice to you, I'm going to approve either way :wink: